typescript/agentkit/src/network/svm.ts

```diff
@@ -0,0 +1,17 @@
+// Chain IDs
+export const SOLANA_MAINNET_CHAIN_ID = 101;
+export const SOLANA_TESTNET_CHAIN_ID = 102;
+export const SOLANA_DEVNET_CHAIN_ID = 103;
+
```

Comment:
are these standard? Is there a more canonical chain identifier that we can use for solana?